openslides/core/static/templates/index.html

```diff
@@ -10,6 +10,7 @@
 <link rel="stylesheet" href="static/css/app.css">
 <link rel="icon" href="/static/img/favicon.png">
 <script src="static/js/openslides-libs.js"></script>
+<script src="js/vendor/angular-pdf/dist/angular-pdf.js"></script>
```

Comment:
Normally this should not be neccessary, because openslides-libs.js should contain all JS files from all bower components if the gulp task was called before.